src/main/java/seedu/address/logic/parser/CliSyntax.java

```diff
@@ -14,5 +14,6 @@ public class CliSyntax {
     public static final Prefix PREFIX_TAG = new Prefix("t/");
     public static final Prefix PREFIX_DATE = new Prefix("d/");
     public static final Prefix PREFIX_CONCERT = new Prefix("c/");
+    public static final Prefix PREFIX_PERSON = new Prefix("p/");
```

Comment:
Does putting the "p/" prefix mean all the other commands which take in a person index have to use this prefix also, or is it just because this command particularly has person index as an optional param

Reply:
It's meant for all other commands too, but I think Karthikeyan is going to edit this in his branch